source/extensions/filters/http/ext_proc/headers.cc

```diff
@@ -0,0 +1,45 @@
+#include "extensions/filters/http/ext_proc/headers.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+using Http::LowerCaseString;
+
+void buildHttpHeaders(const Http::HeaderMap& headers_in,
+                      envoy::config::core::v3::HeaderMap* headers_out) {
+  headers_in.iterate([headers_out](const Http::HeaderEntry& e) -> Http::HeaderMap::Iterate {
+    auto new_header = headers_out->add_headers();
+    new_header->set_key(std::string(e.key().getStringView()));
+    new_header->set_value(std::string(e.value().getStringView()));
+    return Http::HeaderMap::Iterate::Continue;
```

Comment:
API level Q: how come we don't provide the ability to be more selective on which headers are sent in the `ext_proc` config?

Reply:
Just trying to keep feature creep to a minimum. That's certainly an enhancement we can add.